subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/RoomConfigPlugin.kt

```diff
@@ -26,15 +31,29 @@ class RoomConfigPlugin : Plugin<Project> {
             target.dependencies.add("testImplementation", "androidx.room:room-testing:$archPersistence")
         }
 
-        target.plugins.withId("kotlin-kapt") {
+        target.plugins.withId("kotlin-kapt") { plugin ->
 
             target.dependencies.add("kapt", "androidx.room:room-compiler:$archPersistence")
 
             target.extensions.getByType<KaptExtension>().run {
                 arguments {
-                    arg("room.schemaLocation", File(target.projectDir, "room-schemas").absolutePath)
+                    arg("room.schemaLocation", File(target.projectDir, "room-schemas-$kaptIncremental").absolutePath)
+                    arg("room.incremental", kaptIncremental)
                 }
             }
+
+            if (kaptIncremental) {
+                target.afterEvaluate { checkIncrementalKaptSupport(target) }
```

Comment:
What are we waiting for? The plugin has already applied.

Reply:
Not actual. Move this check from this PR to another task.